source/Plugins/SymbolFile/DWARF/DWARFASTParserSwift.cpp

```diff
@@ -119,8 +119,11 @@ lldb::TypeSP DWARFASTParserSwift::ParseTypeFromDWARF(const SymbolContext &sc,
   }
 
   if (!compiler_type && name) {
-    if (name.GetStringRef().startswith("$swift.") ||
-        name.GetStringRef().startswith(SwiftLanguageRuntime::GetCurrentMangledName("_TtBp").c_str())) {  // This is the RawPointerType, need to figure out its name from the AST.
+    if (name.GetStringRef().startswith("$") ||
+        name.GetStringRef().startswith(
+            SwiftLanguageRuntime::GetCurrentMangledName("_TtBp")
```

Comment:
Should we switch to the new mangled name here? 
Also, isn't the second condition made redundant by the first? All the mangled names now start with `$S`

Reply:
It's possible that this is just dead code.

AFAIK all remaining occurrences of _T types in LLDB are hard-coded typenames in the Objective-C Foundation framework that are reserved for Swifty interfaces vended by various of Foundation classes. In this particular case though, I am not so sure.